scripts/data_overlap/common/arguments.py

```diff
@@ -13,7 +13,7 @@ def get_data_overlap_args() -> Any:
         required=True,
         help="The format of your input file for your training data, e.g. raw, custom, the_pile",
     )
-    parser.add_argument("--output-ngrams", type=bool, default=False, help="Whether to output ngrams")
+    parser.add_argument("--output-ngrams", type=bool, default=True, help="Whether to output ngrams")
```

Comment:
If you set the default to True, there's no way to turn it off, right?

Reply:
Thanks for the catch, I've updated it to be `--no-output-ngrams` which can be passed to not output ngrams.